packages/superset-ui-core/test/callApi/callApi.test.js

```diff
@@ -0,0 +1,156 @@
+/* eslint promise/no-callback-in-promise: 'off' */
+import fetchMock from 'fetch-mock';
+import callApi from '../../src/callApi/callApi';
+
+import { LOGIN_GLOB } from '../fixtures/constants';
+import throwIfCalled from '../utils/throwIfCalled';
+
+describe('callApi', () => {
```

Comment:
nit
Add `()` in `describe('callApi()'`) so we know it is a function when reading test result